### PR TITLE
Update Pallet placeholder struct as per FRAME v2

### PR DESCRIPTION
**Pull request by abhi3700**
In the entire codebase, this change has been done as per FRAME v3:
```diff
- pub struct Pallet<T>(PhantomData<T>);
+ pub struct Pallet<T>(_);
```

That's it!

### CODE REVIEW

File: frame/asset-conversion/src/lib.rs [hunk 1]
```diff
@@ -983,7 +983,7 @@ pub mod pallet {
 			let reserve_out = T::HigherPrecisionBalance::from(*reserve_out);
 
 			if reserve_in.is_zero() || reserve_out.is_zero() {
-				return Err(Error::<T>::ZeroLiquidity.into())
+				return Err(Error::<T>::ZeroLiquidity.into());
```
Comment (ggwpez) [inline frame/asset-conversion/src/lib.rs:986]: The fmt CI check is not happy.  
Could you please run `cargo +nightly fmt`?

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable-int
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2970637

**Review by kianenigma — APPROVED** [review]
Once formatting is fixed, thanks!

**Review by ggwpez — APPROVED** [review]
Thanks!

**Comment by ggwpez** [comment]
bot merge

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable-int
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2970979

**Comment by ggwpez** [comment]
@abhi3700 please fix the CI errors, eg:  
```pre
error: unused import: `marker::PhantomData`
    --> frame/contracts/src/lib.rs:132:26
     |
 132 | use sp_std::{fmt::Debug, marker::PhantomData, prelude::*};
```

**Review by sam0x17 — APPROVED** [review]

**Comment by abhi3700** [comment]
> 

Ok will fix it.

**Comment by ggwpez** [comment]
bot merge

**Pull request merged**